CHANGELOG.md

```diff
@@ -2,7 +2,23 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [4.0.7](https://github.com/paritytech/substrate-api-sidecar/compare/v4.0.6...v4.0.7) (2021-04-19)
```

Comment:
Add this line back below since we already released 4.0.7.

Reply:
Didnt even realize i had deleted that... 
Thanks